internal/tiltfile/tiltfile_state.go

```diff
@@ -204,6 +206,16 @@ to your Tiltfile. Otherwise, switch k8s contexts and restart Tilt.`, kubeContext
 	}
 	manifests = append(manifests, localManifests...)
 
+	// if none were requested, then default to all
+	if requested == nil {
```

Comment:
ah, i see in a later PR you changed this to "args" which is GOOD.

I'd clarify this comment--the subject is unclear. maybe something like:
> If no args were specified, i.e. user requested no resources via args, then default to all resources. (Resources specified in Tiltfile override resources specified via args.)

Reply:
Hm, I agree with your comment, but I don't think it's applicable to master, since this code changed.  But, then, your comment is referring to post-later-PR, so I'm not sure how to interpret it.